book/P1C6_survtsk.qmd

```diff
@@ -10,119 +10,270 @@ abstract: "This chapter introduces the different survival problems and formalise
+Another possibility is to instead report the median survival time, but this is only defined if the survival curves drop below $0.5$ within the observed period, which is not guaranteed [@Haider2020].
 
-Predicting survival times is the *deterministic survival task*, defined by predicting a continuous value in the positive Reals and is specified by $g: \calX \rightarrow \NNReals$.
-See @sec-conclusions-faq for practical discussion around predicting in $\NNReals$ vs. $\PReals$ and continuous vs discrete time representations.
-Formally, whilst this is a special case of the ranking task with $\calR \subseteq \NNReals$, the distinction is important as a 'deterministic' prediction specifically refers to forecasting a single determined outcome with a meaningful interpretation, whereas the 'ranking' task is not a deterministic forecast of an event.
+One alternative is to estimate the *restricted mean survival time* (RMST) [@han.restricted.2022; @andersen.regression.2004].
+In contrast to @eq-darth, which integrates the survival curve over the entire time axis, the RMST places an upper-bound on the integral:
 
-<!-- FIXME add example -->
+$$
+\RMST(\tau) = \int^\tau_0 S_Y(y) \ \dy
+$$ {#eq-rmst}
+
+Clearly then $\RMST(\infty) = \EE[Y]$ by @eq-darth.
+Whereas @eq-darth represents the average survival over $[0,\infty)$, @eq-rmst represents the average survival time up to $\tau$.
+Equivalently, the RMST is the average amount of time each individual spends without experiencing the event up to $\tau$.
+The RMST treats all events happening after $\tau$ as if they happened at $\tau$.
+Hence, it is a truncated expectation, estimating: $\EE[\min(Y, \tau)]$
+
+The RMST can have several uses when comparing groups but here its relevant for its ability to give a meaningful lower-bound estimate of $\EE[Y]$ in the presence of censoring.
+For example, say individuals are observed over times $[0,100]$ and administrative censoring is present in the data so that $\hatS(\infty) > 0$ and $\EE[Y]$ cannot be reliably computed.
+However, one can compute $\RMST(100)$ which gives a lower-bound estimate for the expected survival: $\RMST(100) \leq \EE[Y]$.
+This avoids assumptions beyond $\tau=100$ and offers an interpretable prediction: "the average survival time is at least $\RMST(100)$" (@fig-survtsk-rmst).
+The RMST avoids the pitfalls of computing the mean and median by remaining valid even when the predicted distribution is improper, provided that $\tau$ is chosen within the range of observed follow-up times.
+
+As a worked example, say for an individual, $i$, we have: $(t, \hatS_i(t)) = (0, 1), (1, 0.8), (2, 0.4), (3, 0.15)$ then (using a Riemann sum approximation for the integral), $\RMST(4) \approx 1 + 0.8 + 0.4 + 0.15 = 2.35$ whereas $\RMST(3) \approx 1 + 0.8 + 0.4 = 2.2$; the former is a better estimate for $\EE[Y]$.
```

Comment:
I don't think I would frame it in terms "better estimate for E", that would imply it's always best to set \tau to last observed time, which is not how it's used in practice